modules/persistent-storage-csi-vsphere-increase-max-vols-per-node-procedure.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-increase-max-vols-per-node_{context}"]
+= Increasing the maximum allowable volumes per node for vSphere
+
+.Prerequisites
```

Comment:
Under prerequisites we also might want to add statement saying, "Access to vCenter and set the parameter pvscsiCtrlr256DiskSupportEnabled to True". cc. @RomanBednar ?

Reply:
Yes this is a hard requirement in order to make this feature work in OpenShift, unfortunately we still don't have a single official statement from vmware. And this is a vcenter setting that we don't control from OpenShift. So we should be just referencing vmware docs, however we have only this github comment right now: https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/2800#issuecomment-1970185163

Reply:
Done.